src/main/ml-modules/services/KoopProvider.sjs

```diff
@@ -712,14 +712,15 @@ function getObjects(req) {
   pipeline = pipeline
     .select(getSelectDef(outFields, columnDefs, returnGeometry));
 
-  const opticResult = Array.from(pipeline.result(null, bindParams))
-  const opticResultCount = opticResult.length
+  const opticResult = pipeline.result(null, bindParams)
```

Comment:
Due to the bug you found, you should change this back to `Array.from(pipeline.result(null, bindParams))`